lib/ancestry/instance_methods.rb

```diff
@@ -198,21 +202,29 @@ def is_root?
     alias :root? :is_root?
 
     def root_of?(node)
-      self.id == node.root_id
+      self.ancestry_id == node.root_id
     end
 
     # Children
 
     def child_conditions
-      self.ancestry_base_class.child_conditions(self)
+      self.ancestry_base_scope.child_conditions(self)
     end
 
     def children
-      self.ancestry_base_class.where child_conditions
+      self.ancestry_base_scope.where child_conditions
+    end
+
+    def ancestry_base_scope
+      query = self.ancestry_base_class
```

Comment:
is there an example about how the `uniqueness` work here?

Reply:
https://github.com/zugata/ancestry/pull/3/files/9cc34cc0449c28d82678e025b0e1f6eb9d11825c#diff-04c6e90faac2675aa89e2176d2eec7d8R134

Essentially, to us, what makes a performance review unique is the cycle + user_id..  When we want to find descendants of a performance review, we need to scope the `performance_reviews` to the performance_review's cycle.